tests/test_visitors/test_ast/test_operators/test_useless_math.py

```diff
@@ -164,3 +164,23 @@ def test_useful_math(
     visitor.run()
 
     assert_errors(visitor, [])
+
+
+@pytest.mark.parametrize('expression', [
+    '1 / other',
+    '1 / 11',
+    '1 / 1.1',
```

Comment:
Please, also check `1 / 1`. It should raise a violation because of the `/ 1` part